app/models/document_storage.rb

```diff
@@ -0,0 +1,17 @@
+class DocumentStorage < MongoidBase
+  belongs_to :document
+
+  field :type, type: String
+
```

Comment:
Add a helper method:

```
def html?
    self.type == HtmlStorage.TYPE
end
```

May need to be `self._type`, you should add a unit test to double check.